stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1935,10 +1909,9 @@ LogicalResult SelectOp::reifyReturnTypeShapes(
 //===----------------------------------------------------------------------===//
 
 LogicalResult SetDimensionSizeOp::verify() {
-  if (auto size = this->getSize().getType().dyn_cast<RankedTensorType>()) {
+  if (auto size = this->getSize().getType().dyn_cast<RankedTensorType>())
     if (size.getRank() != 0)
       return emitOpError() << "size operand should be of rank-0";
```

Comment:
```
// Use braces on the outer `if` to avoid a potential dangling `else`
// situation.
if (isa<VarDecl>(D)) {
  if (shouldProcessAttr(A))
    handleAttr(A);
}
```